Lib/test/test_generators.py

```diff
@@ -268,6 +268,84 @@ def loop():
         #This should not raise
         loop()
 
+
+class ModifyTest(unittest.TestCase):
```

Comment:
question: is this a `@cpython_only` test or is this something that others should also support?

Reply:
All Pythons should pass the equivalent test, but the ".0" name is a CPython implementation detail.

I'd leave this as a general test for now. Other implementations can adjust the name if needed.